helpers/constants.py

```diff
@@ -240,11 +243,16 @@ def add_role(self, name):
     ARB_TRICRYPTO_3,
     ARB_SWAPR_WBTC_WETH,
     ARB_SWAPR_WETH,
-    BCVX,
     ETH_TRICRYPTO_1,
     HARVEST_RENCRV,
 ]
 
+NO_BOOST = DISABLED_VAULTS + [
+    BCVX,
+    BVECVX,
+    BBVECVX_CVX,
```

Comment:
Add in convex vaults having emissions removed here:
renBTC
sBTC
tBTC
hBTC
bBTC
pBTC
oBTC
byWBTC
wBTC/ETH sushi (arb and eth)
ibBTC sushi

Reply:
The constants file is getting very unwieldy/large perhaps we should split out the addresses into something easier to use? Probably out of scope of this pr

Reply:
Created #294 to track this